src/sentry/static/sentry/app/components/quickTrace/index.tsx

```diff
@@ -446,3 +452,62 @@ function StyledEventNode({text, hoverText, to, onClick, type = 'white'}: EventNo
     </Tooltip>
   );
 }
+
+type MissingServiceProps = Pick<QuickTraceProps, 'anchor' | 'organization'>;
+type MissingServiceState = {
+  hideMissing: boolean;
+};
+
+const HIDE_MISSING_SERVICE_KEY = 'quick-trace:hide-missing-services';
+
+function readMissingServiceState() {
+  const value = localStorage.getItem(HIDE_MISSING_SERVICE_KEY);
+  return value === '1';
+}
+
+class MissingServiceNode extends React.Component<
+  MissingServiceProps,
+  MissingServiceState
+> {
+  state: MissingServiceState = {
+    hideMissing: readMissingServiceState(),
+  };
+
+  dismissMissingService = () => {
+    const {organization} = this.props;
+    localStorage.setItem(HIDE_MISSING_SERVICE_KEY, '1');
+    this.setState({hideMissing: true});
+    trackAnalyticsEvent({
+      eventKey: 'quick_trace.hide.missing-service',
+      eventName: 'Quick Trace: Missing Service Clicked',
+      organization_id: parseInt(organization.id, 10),
+    });
+  };
+
+  render() {
+    const {hideMissing} = this.state;
+    const {anchor} = this.props;
+    if (hideMissing) {
+      return <React.Fragment />;
+    }
+    return (
+      <React.Fragment>
+        <TraceConnector />
+        <DropdownLink
+          caret={false}
+          title={<EventNode type="white">???</EventNode>}
+          anchorRight={anchor === 'right'}
+        >
+          <DropdownItem first width="small">
+            <ExternalDropdownLink href="https://docs.sentry.io/platforms/javascript/performance/connect-services/">
```

Comment:
Will this only be available for JavaScript projects?

Reply:
Hmm good point, this can appear for any project, I'll backlog something to add a product level doc since connecting service isn't JavaScript specific :+1: